src/v/kafka/server/connection_context.h

```diff
@@ -180,16 +175,23 @@ class connection_context final
             // anonymous user
             return security::acl_principal{security::principal_type::user, {}};
         };
+
+        // authorization disabled?
+        if (!_enable_authorizer) {
+            return security::auth_result::authz_disabled(
+              get_principal(), security::acl_host(_client_addr), name);
+        }
```

Comment:
any particular reason this moved down here?

Reply:
Need access to `get_principal`

Reply:
ahh, missed that